db/migrations/003_create_table_user_bonus.up.sql

```diff
@@ -0,0 +1,42 @@
+-- drop table if exists user_order;
+-- create table user_order (
+--                             id int primary key generated always as identity,
+--                             user_id int references usr(id),
+--                             number varchar(255) unique not null,
+--                             status order_status not null,
+--                             accrual int default 0,
+--                             created_at date not null default now()
+-- );
+
+-- drop table if exists user_bonus;
+--
+-- create table user_bonus (
+--     user_id int references usr(id),
+--     order_id int references user_order(id),
+--     amount int not null default 0,
+--     processed_at date not null default now()
+-- );
+
+-- create index ixc_user_bonus_user_id on user_bonus (user_id);
+-- cluster user_bonus using ixc_user_bonus_user_id;
+
+drop table if exists user_bonus_balance;
+create table user_bonus_balance (
```

Comment:
```suggestion
create balance (
```
очень сложное называние. Что это баланс пользователя вроде итак ясно, других у нас и нет. bonus - обман т.к. тут есть и withdrawn и другие данные,